database/languages/javascript.json

```
@@ -75,5 +75,12 @@
     "url": "https://www.javatpoint.com/javascript-tutorial",
     "category": "languages",
     "subcategory": "javascript"
+  },
+  {
+    "name": "javascript Beginning To Mastery",
+    "description": "This playlist makes you a advance javascript programmer. Teach everything from beginning to advance level in Hindi.",
+    "url": "https://youtube.com/playlist?list=PLwgFb6VsUj_n15Cg_y2ULKfsOR1XiQqPx&si=UJyXUgEH0_94SpYj",
+    "category": "languages",
+    "subcategory": "javascript"
   }
```

[proofreading] As mentioned in the guidelines [here](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md#appending-new-links-), you need to add a `language` field in your source is not in English.  
```suggestion
    "name": "javascript Beginning To Mastery",
    "description": "This playlist makes you a advance javascript programmer. Teach everything from beginning to advance level in Hindi.",
    "url": "https://youtube.com/playlist?list=PLwgFb6VsUj_n15Cg_y2ULKfsOR1XiQqPx&si=UJyXUgEH0_94SpYj",
    "category": "languages",
    "subcategory": "javascript",
    "language": "hindi"
  }
```